backend/test/test_no_waste.py

```diff
@@ -0,0 +1,34 @@
+from no_waste import no_waste
+from no_waste.no_waste import Recipe, Product
+
+
+import pytest
+
+@pytest.fixture
+def client():
```

Comment:
This function is not used anywhere

Reply:
It's injected as parameter to test_product